spec/features/integration_spec.rb

```diff
@@ -87,12 +89,28 @@ def writable_example_id(example)
       example.id[2..].tr("/", "-")
     end
 
-    def assert_command_output_consistent(command, example, success:, worker_status_code: 1, test_stderr_empty: true)
+    def assert_command_output_consistent(command, example, success:, worker_status_code: 1, test_stderr_empty: true, expect_manifest_generation: true)
```

Comment:
😅 this is starting to feel like the wrong abstraction ... we add a parameter for testing two example cases...